assets/js/modules/analytics/dashboard-details/dashboard-details-widget-top-acquisition-sources.js

```diff
@@ -17,14 +17,16 @@
  */
 
 /**
- * Internal dependencies
+ * External dependencies
```

Comment:
Let's remove this comment as there are no external dependencies here.

Reply:
I'm confused; why are these components considered "external"? They are internal to the project and only arranged to "look like" non-project dependencies via the webpack config. What's the thinking behind labelling them external when they're inside the project's source code?

Reply:
Historical context: originally the distinction was some components were part of the module (eg analytics here) and some were "common" modules provided by the core plugin (in the original implementation, modules were structurally more distinct from the core plugin). These were called "external" (to the module) dependencies, and imported using the `GoogleComponents` or `GoogleUtils` prefix (which is an alias in webpack).